n3fit/src/n3fit/layers/rotations.py

```diff
@@ -91,6 +91,28 @@ def call(self, pdf_raw):
         # Concatenating destroys the batch index so we have to regenerate it
         return op.batchit(ret)
 
+class AddPhoton(MetaLayer):
+    """
+    Changes the value of the photon to non-zero.
+
+    Both input and output have shape (1, None, 14). However,
+    the input has a photon which is set to zero, while for the
+    output it is set to its real value.
+    """
+
+    def __init__(self, pdf_ph, output_dim, **kwargs):
+        # pdf_ph must be a tensor of shape (1, xgrid)
+        self.pdf_ph = pdf_ph
+        self.output_dim = output_dim
+        super().__init__(**kwargs)
+    
+    def call(self, pdfs):
+        pdf_list = [self.pdf_ph] + [pdfs[:,:,i] for i in range(1,self.output_dim)]
+        # all the elements of the list must have the same
+        # shape, i.e. (1, xgrid), so that op.stack returns
+        # a tensor of shape (1, xgrid, 14)     
+        return op.stack(pdf_list, axis=2)
```

Comment:
```suggestion
        return op.concatenate([self.pdf_ph, pdfs[:,:,1:]], axis=-1)
```

(but I actually believe the `pdf` should have only 13 elements and this class should always stack the photon=0 on top, which can be a photon=something else in the fiatlux case)